opentimelineio/adapters/cmx_3600.py

```diff
@@ -36,9 +36,15 @@
 import os
 import re
 import math
+import collections
 
 import opentimelineio as otio
 
+
+class EDLParseError(otio.exceptions.OTIOError):
```

Comment:
Should we make a more specific subclass of `OTIOError` along the lines of "AdapterParseError"?

Reply:
Sure. I've been adding adapter-specific exception classes like this, so having a superclass to those makes sense.